posthog/migrations/0385_exception_autocapture_off_for_all.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.19 on 2024-01-12 13:16
+
+from django.db import migrations
+
+
+def switch_exception_autocapture_off_for_all(apps, schema_editor):
+    # We can't import the Person model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Team = apps.get_model("posthog", "Team")
+    teams_with_exception_autocapture_on = Team.objects.filter(autocapture_exceptions_opt_in=True)
+    teams_with_exception_autocapture_on.update(autocapture_exceptions_opt_in=False)
```

Comment:
> automatically running SQL in prod ---> risky

do we need a unit test here? I am always afraid of `updates w/o where` :D
I know the where is above using `filter` but you know...ORMs sometimes mess up as well.

Reply:
Weeeellll, we're setting it to the safe default, so even if it's wrong it's safe to apply
And to a not very large table for a very fast operation, so even if we lock it, we won't lock it for long

(these are words that strike me as easy to hit me over the head with in future... we do have a way to write tests for this... hang on)

Reply:
added a test... I'll merge on Monday morning rather than Friday afternoon since it's not a change I make frequently

interestingly the test should have snapshotted the SQL I thought 🤔